src/tree/help/HelpsTreeItem.ts

```diff
@@ -42,7 +43,19 @@ export class HelpsTreeItem extends AzExtParentTreeItem {
             'https://aka.ms/helppanel_docs',
             getThemedIconPath('book')
         );
-        node.id = '1';
+        node.id = '0';
+
+        return node;
+    }
+
+    private get watchVideosTreeItem(): AzExtTreeItem {
+        const node = new OpenUrlTreeItem(
+            this,
+            localize('views.help.watchVideos', 'Docker Extension Videos'),
+            'https://aka.ms/helppanel_videos',
```

Comment:
Um, should we make the aka.ms link name more specific to the Docker extension?

Reply:
Probably, but the same applies to all the links here. Not sure how much it really matters though.

Reply:
Agree, the others are not extension specific.